app/src/main/res/layout/splash_screen.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please use ConstraintLayout here

Reply:
@jordanrafalm jak użyjesz ConstraintLayout to będziesz mógł za pomocą bias procentowo ustawić elementy na wysokość